### DISCUSSION TITLE
How to create Lw_railway

### DISCUSSION THREAD

**Discussion by IsotoCedex** [Universite-Gustave-Eiffel/NoiseModelling#624] [Q&A]
I´m not able to generate Lw_railway from source data: [SourceData.zip](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/13214206/SourceData.zip)

When run the script "Railway_Emission_from_Traffic", no power emission is calculated. Every LW, for every frecuency, is "-99.99"

**Comment by ochiello** [comment]
Hi IsotoCedex
The file TRAINTYPE of the table RAIL_TRAFFIC must be a Varchar and point to a vehicle type present in the .JSON file in /noisemodelling-emission/src/main/resources/org/noise_planet/noisemodelling/emission/railway/RailwayTrainsets.json.

**Reply by IsotoCedex** [reply]
Thank you @ochiello,

Now it works. The [instructions](https://noisemodelling.readthedocs.io/en/latest/Input_railways.html) lead to error, since they indicate another [table](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/blob/4.X/noisemodelling-emission/src/main/resources/org/noise_planet/noisemodelling/emission/Rail_Train_SNCF_2021.json).

I leave here the data ([TrainsNM.zip](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/13238886/TrainsNM.zip)) of the model that I have calculated, in case someone serves as a guide. Please keep in mind that traffic and speeds are invented. The result is a ridiculously high noise level, but the process is valid.

![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/f07b5bdd-fedb-49f7-9d07-8768c062d7b2)

**Comment by ochiello** [comment]
The high levels may be explained by some of your data !

-  A very high traffic since Tevening(84), Tday(28) and Tnight(14) are **Hourly average** train count. (you have divide by the number of hours for each period)
-  A curve section  300 m < R < 500 m
